### DISCUSSION TITLE
Prisma and Planet Scale branch issue

### DISCUSSION THREAD

**Discussion by jvrdom** [planetscale/discussion#275] [Q&A] [answered]
So, I'm kinda confuse with the branches and push.

I have a `main` branch on planet scale that is a production one.

I create a new one with the CLI: 

`pscale branch create futstats add-subtitle-to-posts`

I made some changes on the schema and connect to the branch:

```
> pscale connect futstats add-subtitle-to-posts --port 3309
Secure connection to database futstats and branch add-subtitle-to-posts is established!.

Local address to connect your application: 127.0.0.1:3309 (press ctrl-c to quit)
```

But when I run `npx prisma db push` I'm having:

```
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": MySQL database "futstats" at "us-east.connect.psdb.cloud:3306"
Error: direct DDL is disabled
   0: sql_migration_connector::apply_migration::migration_step
           with step=AlterTable(AlterTable { table_ids: Pair { previous: TableId(3), next: TableId(3) }, changes: [DropColumn { column_id: ColumnId(31) }] })
             at migration-engine/connectors/sql-migration-connector/src/apply_migration.rs:21
   1: sql_migration_connector::apply_migration::apply_migration
             at migration-engine/connectors/sql-migration-connector/src/apply_migration.rs:10
   2: migration_core::state::SchemaPush
             at migration-engine/core/src/state.rs:349
```

For what I undestand this is because I'm pushing to the main branch?

Checked [here](https://github.com/prisma/prisma/issues/7385) and [here](https://planetscale.com/docs/tutorials/automatic-prisma-migrations) but I'm having the same issue, maybe I'm doing it wrong and something is missing?

Thanks :)

**Comment by janpio** [comment, marked as answer]
Yes, the error message show you are pushing to `us-east.connect.psdb.cloud:3306`. 
To talk to the branch you created manually and connected to locally, you need to update your database URL to be `127.0.0.1:3309` (as shown in the `pscale connect` output).

**Comment by jvrdom** [comment]
@janpio thanks for the response :)

So, I have to manually change the `DATABASE_URL` on my `.env` file to `127.0.0.1:3309`?

Thanks :)

**Reply by janpio** [reply]
Yes, you have to make Prisma talk to the correct database by making sure the correct connection string is used. If you are using `.env`, then that is the way to go.

**Reply by jvrdom** [reply]
There is a way to add it automatically? to know Prism to point to the dev branch when running `npx prisma db push`?
